rllib/utils/tests/test_actor_manager.py

```diff
@@ -368,7 +368,7 @@ def test_tags(self):
         manager.foreach_actor_async(lambda w: w.call(), tag="call")
         time.sleep(1)
         results_ping_pong = manager.fetch_ready_async_reqs(
-            tags="pingpong", timeout_seconds=5
```

Comment:
did anything change that made the timeout constraint worse?

Reply:
Good question. I'm not sure either tbh. The test passes just fine on my laptop.

Reply:
This makes me a bit worried that we have some un-identified regressions in the our core API Actor Manager. The regression happened after some known range of PRs one of which is the deprecation of rllib-contrib algorithms. This test might be covering some bug / perf regression. Can you do a bit more digging into this? It will be hard to debug later otherwise. 

![Uploading image.png…]()

Reply:
Yes, agreed. This sudden flakiness of this test (on the CI) is concerning. I will take a closer look ...